src/Http/Action/Security/LogInAction.php

```diff
@@ -61,6 +61,8 @@ public function __invoke(HttpFoundation\Request $request): HttpFoundation\Respon
                 $request->get('email'),
                 $request->get('password')
             );
+
+            $user = $this->authentication->user();
```

Comment:
Can we move this line out of the `try`-and-`catch` block, perhaps just before we actually need the user?

Reply:
authentication->user() could theoretically an exception (hence the new exception I'm catching further down), hence putting this inside the try-catch. Storm yelled at me when I put it outside the try-catch, where you're suggesting.